chatcommands.py

```diff
@@ -557,6 +557,23 @@ def errorlogs(count):
     return fetch_lines_from_error_log(count or 50)
 
 
+@command(privileged=True, aliases=["ms-status", "ms-down", "ms-up"], give_name=True)
+def metasmoke(alias_used):
+    if alias_used in {"metasmoke", "ms-status"}:
```

Comment:
Could we make these aliases non-privileged so anybody can check the status?

Reply:
Done in 7cb88502682d883862e42b17a670fd577287893a